### ISSUE TITLE
Create 2016-2017 Year and Update Curriculum

### ISSUE THREAD

**Issue by smiths** [smiths/MEASURE#74]
We are getting close to being able to enter data for 2016-2017.  This means we will have to create this year in MEASURE.  Once this is done the curriculum for each program will have to be updated.  Either @zhengm2 can make the changes, if there are not too many, or preferably we teach the department reps how to do it themselves.  This should be discussed in our meeting on the 12th of October (2016).

**Comment by smiths** [comment]
Hello Michelle.  This topic was discussed in the October 12th meeting by Joe, Maria and me.  We are planning on rolling MEASURE out to the departmental contributors for the first week of November.  We will have a meeting with the departmental contributors where you will train them to update their programs, update their curriculum maps and update their measurement maps.  They will be given 2 weeks to update the data.  The roll out to instructors is tentatively taking place the third week of November.

Please create the 2016-2017 year in MEASURE and assign it as the default term.  The instructions you need to follow should be in the admin guide.  If there are any problems, please let me know.  We will also have to update the documentation for the departmental contributors.  I'll create a separate issue for this.

Once you have created the 2016-2017 year, assigned the default term and made any obvious changes to the curriculum, you can close this issue.

**Comment by zhengm2** [comment]
Hi Spencer. Once the new year has been created can I assume that the current year can be archived? The instructions on how to set the default term are not in the guide but I will try it and report any issues.

**Comment by smiths** [comment]
Once we are at steady state, you are correct that we would archive the current year and then move to a new year.  I don't think we should archive it yet though.  I don't think we will be adding much more data, but we might.  We can archive it later.

I didn't know that the default term instructions were not in the guide.  The instructions should be very similar to setting the default year.  If you run into any trouble, please let me know.  We'll need to make sure that the documentation is updated to include the default term.

**Comment by zhengm2** [comment]
Hi Spencer, 2016-17 has been created and I have set 2016-17 Term 1 as the default term (yes this was included in the instructions). I am working on the the curriculum changes.

**Comment by smiths** [comment]
Sounds good.  Thank you Michelle.
